test/e2e/image_sign_test.go

```diff
@@ -0,0 +1,63 @@
+// +build !remote
+
+package integration
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	. "github.com/containers/podman/v2/test/utils"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Podman image sign", func() {
+	var (
+		tempdir    string
+		err        error
+		podmanTest *PodmanTestIntegration
+	)
+
+	BeforeEach(func() {
+		tempdir, err = CreateTempDirInTempDir()
+		if err != nil {
+			os.Exit(1)
+		}
+		podmanTest = PodmanTestCreate(tempdir)
+		podmanTest.Setup()
+		podmanTest.SeedImages()
+
+		tempGUNPGHOME := filepath.Join(podmanTest.TempDir, "tmpGPG")
+		err := os.Mkdir(tempGUNPGHOME, os.ModePerm)
+		Expect(err).To(Equal(nil))
+
+		origGNUPGHOME := os.Getenv("GNUPGHOME")
+		defer os.Setenv("GNUPGHOME", origGNUPGHOME)
```

Comment:
This is called at the end of the `BeforeEach()` callback; it needs to happen in `AfterEach` instead.